tests/unit/isReportMessageAttachmentTest.ts

```diff
@@ -1,16 +1,17 @@
+import type {Message} from '@src/types/onyx/ReportAction';
 import isReportMessageAttachment from '../../src/libs/isReportMessageAttachment';
 
 describe('isReportMessageAttachment', () => {
     it('returns true if a report action is attachment', () => {
-        const message = {
+        const message: Message = {
```

Comment:
Should we be adding a `type` in the test instead of making it optional? Do we know that type might not exist?